lib/app/config/logger.dart

```diff
@@ -11,6 +12,13 @@ class Log {
 
   static void w(String message) => logger.w(message); // coverage:ignore-line
 
-  static void e(String message, Object error, {StackTrace? stackTrace}) =>
-      logger.e(message, error: error, stackTrace: stackTrace);
+  static void e(
+    String message,
+    Object error, {
+    StackTrace? stackTrace,
+    FirebaseCrashlytics? crashlytics,
+  }) {
+    logger.e(message, error: error, stackTrace: stackTrace);
+    if (crashlytics != null) crashlytics.recordError(error, stackTrace, fatal: true);
```

Comment:
?.
Inside of if not no

Reply:
Sorry, I didn't quite understand. Do you want me to use block body {} instead of one line?